packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +155,8 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    ++this.readStackEventsCalls;
+    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME + (this.readStackEventsCalls * CFN_POLL_TIME_BACKOFF));
   }
```

Comment:
If you want the interval to incrementally backoff as `this.readStackEventsCalls` is incremented, shouldn't the interval function be responsible for incrementing `this.readStackEventsCalls`?

Reply:
I'm assuming that addToPollQueue is queuing up calls to describeStack for stackName; 
`addToPollQueue(stackId, priority = 5) {
    if (!this.pollQueueStacks.includes(stackId)) {
      this.pollQueueStacks.push(stackId);
      this.pollQueue.schedule({ priority }, () => {
        this.removeFromPollQueue(stackId);
        return this.pollStack(stackId);
      });
    }
    return false;
  }`

Reply:
A refactor would be required since the interval is set in this function. I'm hoping to limit any behavioral changes to just the polling delay in this PR.